apitests/spec/devices_spec.ts

```diff
@@ -259,7 +291,7 @@ describe('devices api', () => {
         });
 
         it('should return command history', (done) => {
-            request.get('/'+enabled_device, (err, resp, result:SingleDevice) => {
+            request.get('/'+device_with_cmdHistory, (err, resp, result:SingleDevice) => {
```

Comment:
nit: camelCase over snake unless your trying to keep to a convention of snake for variables and camel for functions. Either way, this is mixing them